docs/support/faq.md

```diff
@@ -158,75 +158,75 @@ For other permissions, please see the contract [here](https://github.com/LIT-Pro
 
-For different resources you’ll be required to sign each time since the signed message is different. One way to get around this is to use a PKP to do the signing for you. This is how it works: The user owns a PKP & uses it to sign a message in a LitAction with `Lit.Actions.signEcdsa()`. Then return all the sigShares from the LitAction to the app. Here the user can loop through & craft AuthSigs for each returned sigShare and process the required access control operation like decryption. Another way is to return the crafted AuthSigs from the LitActions directly.
+For different resources, you'll be required to sign each time since the signed message is different. One way to get around this is to use a PKP to do the signing for you. This is how it works: The user owns a PKP & uses it to sign a message in a LitAction with `Lit.Actions.signEcdsa()`. Then return all the sigShares from the LitAction to the app. Here the user can loop through & craft AuthSigs for each returned sigShare and process the required access control operation like decryption. Another way is to return the crafted AuthSigs from the LitActions directly.
 
 ### 5. Out of the above two approaches, which one is preferred?
 
-A key point to note here is that the LitAction can execute only for 60 seconds after which it times out. Hence it makes sense to reduce complexity in the LitActions code. Also if you want to create a lot of AuthSigs process those in batches. This will require multiple calls to execute the LitAction. You may also parallelize these promises using a package like [p-queue](https://www.npmjs.com/package/p-queue).
+A key point to note here is that the LitAction can execute only for 60 seconds after which it times out. Hence it makes sense to reduce complexity in the LitActions code. Also, if you want to create a lot of AuthSigs, process those in batches. This will require multiple calls to execute the LitAction. You may also parallelize these promises using a package like [p-queue](https://www.npmjs.com/package/p-queue).
 
 <br />
 
 ## Security & Trust Implications
 
 ### 1. What encryption algorithm are you using? AES?
 
-Yes. AES-GCM webcrypto for the symmetric encryption. then that key is encrypted to the Lit network's BLS public key. the BLS private key shares are used by the nodes to decrypt.
+Yes, AES-GCM webcrypto is used for symmetric encryption. Then that key is encrypted to the Lit network's BLS public key. The BLS private key shares are used by the nodes to decrypt.
 
-### 2. How does Lit handles key management?
+### 2. How does Lit handle key management?
 
-There is only one key, created with distributed key generation. The nodes all know the public key but nobody knows the whole private key.
+There is only one key, created with distributed key generation. The nodes all know the public key, but nobody knows the whole private key.
```

Comment:
this isn't true, there are multiple root keys which are used to generate PKPs

Reply:
Rectified @a1ttech !